### PR TITLE
Improve description and preferences

### LINKED ISSUE #68: nit: Improve extension description
The current extension description is a bit wordy:
`"This extension provides a Red Hat OpenShift Local cluster and allows Podman Desktop to interact with it"`
and stands out from the other extensions. It should probably be something more like:
`"Integration for OpenShift Local"`
or at least start with that and include a short description of what OpenShift Local is or what you can do with it.

### PR DESCRIPTION
**Pull request by deboer-tim**
Minor improvements to the extension description and preferences:
- Updated extension description. It is a bit bland, but shorter/concise and matches other extensions more.
- Moved memory minimum from text to be enforced (like disk size is), and made the two descriptions consistent and match other preferences.
- Used markdown to change link to pull secret into a real link. Easier to copy for now; will be clickable once [1] is fixed.

Fixes #68.

[1] https://github.com/containers/podman-desktop/issues/2230

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
         "OpenShift-Local.memory": {
           "type": "number",
           "format": "memory",
-          "description": "Memory size in MiB (must be greater than or equal to '2048')"
+          "minimum": 2048,
```
Comment (gbraad) [inline package.json:27]: We can not have a minimum stated in the config, as that also means this is the minimum for the OpenShift preset, which is not true. Best to not mention this and let the UI handle the lower boundary

**Review by gbraad — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "openshift-local",
```
[the remark this answers is not in the document]
Comment (gbraad) [reply package.json:2]: Creating a new issue for PD

[the remark this answers is not in the document]
Comment (deboer-tim) [reply package.json:2]: I just checked and confirmed it is using `name`, and agree it should use the `display name`.

File: package.json (hunk 1 above)
Comment (deboer-tim) [reply package.json:27]: I'm not sure I follow, because that sounds like exactly what I thought I did. :) Do you mean not mention nor enforce the minimum here and handle it elsewhere?

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (gbraad) [reply package.json:2]: I created: https://github.com/containers/podman-desktop/issues/2358

File: package.json (hunk 1 above)
Comment (gbraad) [reply package.json:27]: we purposely removed this value from package json as this should be controlled by the extension directly: it is based on the preset. using it this way from the package.json therefore conflicts

Comment (gbraad) [reply package.json:27]: The removal of `memory` happened in: https://github.com/crc-org/crc-extension/commit/b43b855f29435a9315de9e4debcdabd44733401d

@evidolob WDYT?

Comment (evidolob) [reply package.json:27]: Yes, we remove all minimums values from package.json, as we cannot modify them on the runtime, and they are changed, based on preset. So we perform validation after user changed some preference from UI.

Comment (gbraad) [reply package.json:27]: Please remove this change from the PR, after which it can be approved.

**Comment by gbraad** [comment]
Superceded by #96
